src/common/libflux/rpc.h

```diff
@@ -22,6 +22,9 @@ flux_future_t *flux_rpc_raw (flux_t *h, const char *topic,
                              const void *data, int len,
                              uint32_t nodeid, int flags);
 
+flux_future_t *flux_rpc_message (flux_t *h, const flux_msg_t *msg,
```

Comment:
do we need a new manpage entry for this function?

Reply:
yes thanks!